packages/rollup-plugin-import-meta-assets/test/integration.test.js

```diff
@@ -258,19 +258,20 @@ describe('rollup-plugin-import-meta-assets', () => {
     );
   });
 
-  it('allows backticks in path if there are no dynamic parts included', async () => {
+  it.only('allows backtics and dynamic vars in path', async () => {
```

Comment:
```suggestion
  it('allows backtics and dynamic vars in path', async () => {
```

Reply:
Will change

Reply:
We really should have a lint rule for this :)